components/ProfilePage/SocialMediaIcons.tsx

```diff
@@ -49,6 +53,22 @@ export const SocialMediaIcons = ({
             </External>
           </Col>
         )}
+
+        {blueSky && (
+          <Col>
+            <External plain href={`https://bsky.app.profile/${blueSky}`}>
+              <Image alt={t("socials.bluesky")} src="/bluesky.svg" />
+            </External>
+          </Col>
+        )}
+
+        {mastodon && (
+          <Col>
+            <External plain href={`https://mastodon.org/${mastodon}`}>
```

Comment:
nit: mastodon link should be `https://mastodon.social/`, not `https://mastodon.org`.

We should also add some light validation to ensure that there is an `@` at the beginning of the mastodon name. We can add a check to `cleanSocialUrls` to remove any leading `@` signs from `mastodon` names, and then just include the `@` sign in the mastodon url (e.g. `https://mastodon.social/@${mastodonName}`)